apps/i18n/common/en_us.json

```diff
@@ -1874,7 +1874,7 @@
   "reset": "Reset",
   "resetProgram": "Reset",
   "resetPassword": "Reset password",
-  "resetTeacherPasswordTooltip": "You cannot reset passwords for teacher accounts in your section.",
+  "secretLoginTooltip": "Secret login disabled for teachers",
```

Comment:
It seems like "Secret login" is how we use this method in code, but "Picture password" is how it appears in the screenshots... do we have a different word for Picture Password and Secret Word login? Or is this what we use user-facing?

Reply:
Yeah, as you've pointed out the user-facing wording is `Picture password` or `Secret Word`. There is a dropdown when setting up a section where the options are `Picture login` and `Word login`. Internally, we call these `secret picture` and `secret words`. The component is called `Show Secret`. So... not super consistent haha!

I think that's what influenced a single message string for both of these types of logins, however I can see how that might be confusing from an external user perspective. I could see two options:

1) Simplify this to be `Disabled for teachers`, as the action of hovering over the button (Show picture/Show words) would be descriptive of "what" is disabled.
2) Change this to two specific messages, "Picture password disabled for teachers", and "Secret words disabled for teachers"

Reply:
Or maybe, `You cannot view passwords for teacher accounts in your section.`